### PR TITLE
chore: merge B14S6 beta to alpha

### PR DESCRIPTION
**Pull request by mcdurdin**

### CODE REVIEW

File: HISTORY.md [hunk 1]
```diff
@@ -72,6 +72,8 @@
 * fix(windows): Track modifier changes in UWP apps (#4468)
 * fix(common/resources): Fix help.keyman.com path for commit (#4469)
 * fix(common): create GitHub comments serially (#4472)
+* feat(common/models): mid-context suggestions & reversions, fix(common/models): correction-search SMP issues (#4427)
```
Comment (jahorton) [inline HISTORY.md:75]: This PR hasn't been merged yet and should not be mentioned.

File: HISTORY.md [hunk 2]
```diff
@@ -213,6 +215,24 @@
...
+* fix(ios): tutorial's link to "Add a Keyboard" links directly to keyboard search (#4491)
+* feat(linux): Improve output of km-package-list-installed (#4481)
+
+## 14.0.242 beta 2021-02-11
+
+* fix(android/engine): Display online keyboard help (#4462)
+* fix(windows): Track modifier changes in UWP apps (#4468)
+* fix(common/resources): Fix help.keyman.com path for commit (#4469)
+* fix(common): create GitHub comments serially (#4472)
+* feat(common/models): mid-context suggestions & reversions, fix(common/models): correction-search SMP issues (#4427)
```
Comment (jahorton) [inline HISTORY.md:233]: #4427 isn't merged yet.

**Review by jahorton — CHANGES_REQUESTED** [review]
A few history.md issues showed up.

File: HISTORY.md (hunk 1 above)
Comment (mcdurdin) [reply HISTORY.md]: Yeah, saw that as I looked at HISTORY.md and then forgot to fix it while fixing up all the versioning merge conflicts yay package.json :(

File: HISTORY.md (hunk 2 above)
Comment (mcdurdin) [reply HISTORY.md]: I think this relates to https://github.com/keymanapp/keyman/pull/4427#issuecomment-777681199... which I am investigating in #4483...

**Review by jahorton — APPROVED** [review]

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 15.0.17-alpha](https://keyman.com/downloads/releases/alpha/15.0.17)

**Pull request merged**